config/routes.rb

```diff
@@ -141,7 +141,7 @@
     get 'search', on: :collection
   end
 
-  resource  :setup, :session, :search, :printable_directory, :privacy
+  resource  :setup, :session, :search, :printable_directory
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.